### ISSUE TITLE
Visualization not advancing past first frame

### ISSUE THREAD

**Issue by sbuch-cpu** [facebookresearch/iSDF#19]
Hello, thanks for publishing this work its potential applications are very interesting.
I've been trying to get the replicaCAD example working as outlined in the README.md file. I am able to download the data and launch the iSDF viewer, however, the process never advances past frame 0 before saying the sequence has ended (20 minutes and a few hundred steps later). I'm wondering if there is some command or button that I'm missing to get it to move onto the next frame?

**Comment by BillyMazotti** [comment]
I have a similar issue. Originally I was also only getting the initial key frame when allocating 4-5 CPU cores, however now I'm getting between 2-3 frames when allocating 8 CPU cores (see images below). 

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/96280520/228610830-addede31-6eb5-491c-b90b-4086cf6a405f.png">

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/96280520/228615823-20a10d43-664f-4237-b731-7bf5cf9211ed.png">

Still, this number of keyframes seems too small. I expect this is a result of the machine I'm using which limits me to only using the cpu version of PyTorch.


My setup is:
- platform: Mac M1 - Ubuntu 20.04.5 using VMware Fusion ([setup guide](https://dev.to/daud99/installing-ubuntu-using-vmware-fusion-tech-preview-on-mac-m1-silicon-4b0e))
- isdf conda environment: [environment_macm1_vmwarefusion_ubuntu20045.yml.zip](https://github.com/facebookresearch/iSDF/files/11103346/environment_macm1_vmwarefusion_ubuntu20045.yml.zip)

Regarding the conda environment, I found the provided environment.yml file gave me ResolvePackageNotFound errors for 20+ packages. I solved this by adding conda-forge as channel in the .yml file, removing channel specifications for all packages, and removing version specifications for all packages besides python, setuptools, pyglet, and pip.
